astrobee/config/communications/dds_ros_bridge.config

```diff
@@ -148,6 +148,16 @@ pub_topic_RLRLS = ""
 use_RORP = true
 pub_topic_RORP = ""
 
+-- ros_sparse_mapping_pose_rapid_position => RSMPRP --
+use_RSMPRP = true
+pub_topic_RSMPRP = "-sparse"
+use_rate_RSMPRP = true
+
+-- ros_vive_pose_rapid_position => RVPRP --
+use_RVPRP = true
```

Comment:
This needs to be changed to false before being merged into develop. It was for testing purposes only.

Reply:
Done